src/index.html

```diff
@@ -1,30 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
- [ ] I also noticed that when the page loads first, the image below is what the user sees first. This I believe is an error. Kindly fix this. If you are not experiencing this in your machine, try to clone the project and serve that newly cloned project. If you still do not experience this, please ignore this comment.

![image](https://user-images.githubusercontent.com/84156714/212487725-23181283-a2df-40d6-8ffc-32c8c1a5e25d.png)